CONTRIBUTING.md

```diff
@@ -135,6 +135,30 @@ Most features will likely have most of the code in the `containerd-shim-wasm` pr
 
 A tip for developing a new feature is to implement it and test it with one runtime you are familiar with then add it to all the runtimes.  This makes it easier to test and iterate before making changes across all the runtimes.
 
+## Adding new shims
+
+We welcome new shims, though you can also host them in your own repositories as well and use the `coantainerd-shim-wasm` crate.  We recognize that the project is moving fast and having them in this repository can reduce the need for changes as well for discoverability.  
+
+Please open an issue before submitting a PR for discussion to make sure it is a good fit.  As a general rule, we want shims to be adopting WASI spec (this is after all called run*wasi* :-)). In the future we may require shims in the repository to pass WASI compliance tests when the standards mature more. See https://github.com/containerd/runwasi/issues/338 for more discussion.
+
+Since we are not experts in every runtime we also need a commitment from the runtime owners to contribute to maintenance of the shim.
+
+## Removing Shims
+
+This is a fast moving space, with lots of innovation happening and some shims may eventually need to be removed.
+
+A Shim implementation maybe subject to removal if:
+ - If a shim runtime its self has not maintained for 6 months it will be subject to removal.  
```

Comment:
Wording here seems a bit off.

```suggestion
 - If a shim runtime has not been maintained for 6 months it will be subject for removal
 ```

Reply:
I also think this should timeframe should be shortened for now given how things are moving.

Reply:
Maybe a callout that the shim may be removed from CI after a certain time period if it is not maintained?
Like 1 month?

Reply:
"subject to" is the proper form; both suggested changes are off by one grammar oddity. :-)

has not been.... subject to.

Reply:
fixed